README.md

```diff
@@ -1,8 +1,33 @@
-Citadel
-=======
+# Argentum Age
 
-A turn based strategy game based on the Anura engine
+[<img src="http://argentumage.com/favicon-32.png" width="20"> Official website](https://argentumage.com) | [<img src="https://cdnjs.cloudflare.com/ajax/libs/webicons/2.0.0/webicons/webicon-twitter.svg" width="20"> Twitter](https://twitter.com/argentum_age) | [<img src="https://cdnjs.cloudflare.com/ajax/libs/webicons/2.0.0/webicons/webicon-reddit.svg" width="20"> Reddit](https://www.reddit.com/r/ArgentumAge/) | [<img src="https://cdnjs.cloudflare.com/ajax/libs/webicons/2.0.0/webicons/webicon-facebook.svg" width="20"> Facebook](https://www.facebook.com/ArgentLark) | [<img src="https://discordapp.com/assets/f8389ca1a741a115313bede9ac02e2c0.svg" width="22"> Discord](https://discord.gg/0s6mSoZ1vJPlvcIw)
```

Comment:
Here a line break would do the same as a single space. So you can break a huge line into five smaller lines to no render disadvantage but easier future diff maintenance.

Reply:
Ok.